unittests/test_kernels.py

```diff
@@ -1,1009 +1,968 @@
+        with pytest.raises(KeyError):
+            srfs2c('whatever', 'phobos')
+        with pytest.raises(KeyError):
+            srfs2c('PHOBOS GASKELL Q512', 'deimos')
+        with pytest.raises(KeyError):
+            srfscc('whatever', 401)
+        with pytest.raises(KeyError):
+            srfscc('PHOBOS GASKELL Q512', 402)
+
+        self.assertAllEqual(srfc2s.flag(1, 401), ['PHOBOS GASKELL Q512', True])
+        self.assertAllEqual(srfcss.flag(1, 'phobos'), ['PHOBOS GASKELL Q512', True])
+        self.assertAllEqual(srfs2c.flag('PHOBOS GASKELL Q512', 'phobos'), [1, True])
+        self.assertAllEqual(srfscc.flag('PHOBOS GASKELL Q512', 401), [1, True])
+
+        assert not srfc2s.flag(2, 401)[1]
+        assert not srfc2s.flag(1, 402)[1]
+        assert not srfcss.flag(2, 'phobos')[1]
+        assert not srfcss.flag(1, 'deimos')[1]
+        assert not srfs2c.flag('whatever', 'phobos')[1]
+        assert not srfs2c.flag('PHOBOS GASKELL Q512', 'deimos')[1]
+        assert not srfscc.flag('whatever', 401)[1]
+        assert not srfscc.flag('PHOBOS GASKELL Q512', 402)[1]
+
+    def test_timdef(self):
+        assert timdef('get', 'calendar') == 'GREGORIAN'
+        assert timdef('get', 'system') == 'UTC'
+        assert timdef('get', 'zone') == ''
+
+        assert timdef('set', 'calendar', 'mixed ') == 'mixed '
+        assert timdef('get', 'calendar') == 'MIXED'
+
+        assert timdef('set', 'system', 'utC') == 'utC'
+        assert timdef('get', 'system') == 'UTC'
+
+        assert timdef('set', 'zone', 'PDT') == 'PDT'
+        assert timdef('get', 'zone') == 'UTC-7'
+        timdef('set', 'zone', 'UTC-0')
+
+    def test_tparse(self):
+        assert tparse('Tue Aug  6 11:10:57  1996') == -107398143.0
+        assert tparse('JANUary 1, 2000 12:00') == 0.
+
+        with pytest.raises(ValueError):
+            tparse('Tue Aug  6 11:10:57  1996g')
+        try:
+            _ = tparse('Tue Aug  6 11:10:57  1996g')
+        except ValueError as e:
+            fullmsg = str(e)
+            msg = fullmsg.split(' -- ')[2]
+
+        assert tparse.flag('Tue Aug  6 11:10:57  1996') == [-107398143.0, '']
+        assert tparse.flag('JANUary 1, 2000 12:00') == [0., '']
+        assert tparse.flag('Tue Aug  6 11:10:57  1996g')[1] == msg
+
+    def test_tpictr_timout(self):
+        time = 'Tue Aug 06 11:10:57  1996'
+        pictr = 'Wkd Mon DD HR:MN:SC  YYYY'
+        secs = -107398143.0
+        assert tparse(time) == secs
+        assert tpictr(time) == pictr
+        assert timout(secs + deltet(secs, 'UTC'), pictr) == time
+
+        time = 'JANUARY 01, 2000 12:00'
+        pictr = 'MONTH DD, YYYY HR:MN'
+        secs = 0.
+        assert tparse(time) == secs
+        assert tpictr(time) == pictr
+        assert timout(secs + deltet(secs, 'UTC'), pictr) == time
+
+        with pytest.raises(ValueError):
+            tpictr('Tue Aug  6 11:10:57  1996g')
+        try:
+            _ = tpictr('Tue Aug  6 11:10:57  1996g')
```

Comment:
Don't need `_ =` (see previous lines)

Reply:
One of the problems of using an automatic code converter.  I caught some of its goofs, but not that on.